kv/lookup_service.go

```diff
@@ -0,0 +1,66 @@
+package kv
+
+import (
+	"context"
+
+	"github.com/influxdata/influxdb"
+)
+
+var _ influxdb.LookupService = (*Service)(nil)
+
+// Name returns the name for the resource and ID.
+func (s *Service) Name(ctx context.Context, resource influxdb.ResourceType, id influxdb.ID) (string, error) {
```

Comment:
comepletely unrelated, but what if we did something like a generic `Find` with something like this

```go
func (s *Service) Find(ctx context.Context, resource influxdb.ResourceType, id influxdb.ID, v interface{})  error {}
```
we could map `resource -> kv.bucket` and lookup the look up the ID of the resource in that bucket.

Reply:
Yeah I think a generic find would be nice for sure.  This refactor has shown me we could have a few functions like that Find that could be used by our implementations.